integration_tests/tests/integrity/ap_ar_enhanced_count_match.sql

```diff
@@ -0,0 +1,48 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with ap_source as (
+    select
+        bill_id,
+        count(*) as bill_row_count
+    from {{ ref('stg_sage_intacct__ap_bill_item') }}
```

Comment:
It is possible `sage_intacct__using_bills` could be set to false, could this cause a compilation error? Do we need to set up a variable config similar to what exists in line 3 of `sage_intact__ap_ar_enhanced`?

Reply:
Great question. This would technically fail if the bill variable is set to false. We could add that in, but I don't feel it is entirely necessary right now. I added a comment to highlight that this test is written under the assumption that the variables are set to true (default config). 

Additionally, I don't feel we need to over engineer this validation. We are primarily trying to confirm that there is no over aggregation or fanout occurring, not that the variables are working as intended (that is what the normal buildkite tests are for). Therefore, since this is only for our internal validations, this should be fine to keep as is but I would be open to expanding the test for variable configs in the future if we wanted.

I am also open to discussing this further with the team during standup since this is a new process.

Reply:
Can discuss during standup, but this makes sense to me for this review.